hazelcast/src/main/java/com/hazelcast/spi/impl/servicemanager/impl/ServiceManagerImpl.java

```diff
@@ -274,7 +274,7 @@ private void shutdownService(final ManagedService service, final boolean termina
         }
     }
 
-    private synchronized void registerService(String serviceName, Object service) {
+    protected synchronized void registerService(String serviceName, Object service) {
```

Comment:
I didn't find a programatic way how to register custom service.
Could you please help here ?